changelog/updates/2021-12-17-eapi4-update.md

```diff
@@ -0,0 +1,8 @@
+- ipset ([7.11](https://ipset.netfilter.org/changelog.html))
```

Comment:
I guess missing packages in the changelog concern the SDK and not the end user ?

Reply:
Yes, the unmentioned packages are not user facing.